tests/Command/Dispatcher/DispatchFilesCommandTest.php

```diff
@@ -75,7 +75,7 @@ public function projectProvider(): iterable
   composer_version: '1'
   phpstan: true
   psalm: true
-  panther: true
+  phpunit_extensions: ['panther', 'doctrine_test']
```

Comment:
Would prefer something like this:

```suggestion
  phpunit_extensions: 
    - Symfony\Component\Panther\ServerExtension
    - DAMA\DoctrineTestBundle\PHPUnit\PHPUnitExtension
```

Reply:
I dont think thats a good idea, the phpunit extensions (in case of panther) are not only used to enable the extension. Panther needs an env var and some gitignore files.